packages/web/components/MegaMenu.tsx

```diff
@@ -281,19 +292,38 @@ const PlayerStats: React.FC<PlayerStatsProps> = ({ player, pSeedBalance }) => (
         {pSeedBalance || 0}
```

Comment:
Currently the pSeedBalance is shown in 18 decimals
There is a helper in `@metafam/utils` called `numbers` which contains `amountToDecimal` to turn it in a readable number
It's also being used in 'web/utils/questHelper.ts' on line 31
Could you use it here?